libcaf_test/caf/test/reporter.cpp

```diff
@@ -112,15 +112,24 @@ struct colorizing_iterator {
           out->put(c);
         break;
       case off_read_color:
-        mode = off_escape;
+        out->flush();
```

Comment:
Why did you add the `flush` here?

Reply:
You have mentioned that off_* will be similar to it's colored states. We are using flush to print everything in read_color state. I think that this behavior needs to be the same in off_read_color state in order to maintain it's parallel to read_color state.